src/main/java/org/broadinstitute/hellbender/utils/help/GATKWDLWorkUnitHandler.java

```diff
@@ -135,58 +143,142 @@ protected void processPositionalArguments(
             final CommandLineArgumentParser clp,
             final Map<String, List<Map<String, Object>>> args) {
         super.processPositionalArguments(clp, args);
-        final List<Map<String, Object>> positionalArgsList = args.get("positional");
+        final List<Map<String, Object>> positionalArgsList = args.get(TemplateProperties.ARGUMENTS_POSITIONAL);
         if (positionalArgsList != null && !positionalArgsList.isEmpty()) {
-            final Map<String, Object> positionalArgs = args.get("positional").get(0);
-            final String argType = (String) positionalArgs.get("type");
-            positionalArgs.put("testValue",
-                    getInputValueForTest(
-                            argType,
-                            (String) positionalArgs.get("defaultValue"))
+            final Map<String, Object> positionalArgs = args.get(TemplateProperties.ARGUMENTS_POSITIONAL).get(0);
```

Comment:
Are you guaranteed that there's always at least one element here?

Reply:
Yes, this code block is inside a guard that checks that there actually are positional args.